src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECDsa.cs

```diff
@@ -8,6 +8,20 @@ namespace System.Security.Cryptography
 {
     public abstract class ECDsa : AsymmetricAlgorithm
     {
+        public ECDsa() { }
+
+        public ECDsa(ECNamedCurve curve)
```

Comment:
This seems weird... we'd potentially want the Curve value even if loading a saved key.

Though since Curve is protected... what's it really doing?